src/state/gameProcessReducer.test.ts

```diff
@@ -0,0 +1,162 @@
+      isBlocking: false,
+      answers: [], // waits for answer as boolean
+      result: '',
+    };
+  });
+  // **************************************************************************
+  it('should return a correct isPlaying boolean value', () => {
+    const isPlayingAction: GameProcessAction = {
+      type: IS_PLAYING,
+      payload: true,
+    };
+    const newState: GameProcessState = {
+      isPlaying: true,
+      isGameStarted: false,
+      isBlocking: false,
+      answers: [], // waits for answer as boolean
+      result: '',
+    };
+    expect(gameProcessReducer(initState, isPlayingAction)).toMatchObject(newState);
+    expect(gameProcessReducer(initState, isPlayingAction)).toStrictEqual(newState);
+  });
+  // *************************************************************************************
+  it('should return a correct isGameStarted boolean value', () => {
+    const isGameStartedAction: GameProcessAction = {
+      type: IS_GAME_STARTED,
+      payload: true,
+    };
+    const newState: GameProcessState = {
+      isPlaying: false,
+      isGameStarted: true,
+      isBlocking: false,
+      answers: [], // waits for answer as boolean
+      result: '',
+    };
+    expect(gameProcessReducer(initState, isGameStartedAction)).toMatchObject(newState);
+    expect(gameProcessReducer(initState, isGameStartedAction)).toStrictEqual(newState);
+  });
+  // *************************************************************************************
+  it('should return a correct isBlocking boolean value', () => {
+    const isBlockingAction: GameProcessAction = {
+      type: IS_BLOCKING,
+      payload: true,
+    };
+    const newState: GameProcessState = {
+      isPlaying: false,
+      isGameStarted: false,
+      isBlocking: true,
+      answers: [], // waits for answer as boolean
+      result: '',
+    };
+    expect(gameProcessReducer(initState, isBlockingAction)).toMatchObject(newState);
+    expect(gameProcessReducer(initState, isBlockingAction)).toStrictEqual(newState);
+  });
+  // *************************************************************************************
+  it('should return a correct answers array with a boolean values', () => {
+    const addAnswerActionTrue: GameProcessAction = {
+      type: ADD_ANSWER,
+      payload: true,
+    };
+
+    const addAnswerActionFalse: GameProcessAction = {
+      type: ADD_ANSWER,
+      payload: false,
+    };
+
+    const newStateFirst: GameProcessState = {
+      isPlaying: false,
+      isGameStarted: false,
+      isBlocking: false,
+      answers: [true],
+      result: '',
+    };
+
+    const newStateSecond: GameProcessState = {
+      isPlaying: false,
+      isGameStarted: false,
+      isBlocking: false,
+      answers: [false],
+      result: '',
+    };
+
+    expect(gameProcessReducer(initState, addAnswerActionTrue)).toMatchObject(newStateFirst);
+    expect(gameProcessReducer(initState, addAnswerActionTrue)).toStrictEqual(newStateFirst);
```

Comment:
would `.toStrictEqual(newStateFirst);` be enough for all  addAnswerActionTrue results?